ndcube/ndcube.py

```diff
@@ -756,6 +756,15 @@ class NDCube(NDCubeBase):
         Default is False.
 
     """
+    # Enabling the NDCube reflected operators is a bit subtle.  The NDCube
```

Comment:
My feeling is that we probably want all these arithmetic methods on `NDCubeBase` rather than `NDCube`.  That way `NDCube` will inherit them.  Are there any arguments against? @wtbarnes @Cadair ?

Reply:
I think we briefly talked about this at the coordination meeting and I *thought* the consensus was that it should go on `NDCube` rather than `NDCubeBase` but perhaps I am mistaken.

Reply:
Oh.  I don't remember that.  Perhaps @Cadair can have the casting vote on which memory is right.

Reply:
After further discussion over :beers: let's leave these on `NDCube` for now as there's no penalty for moving them to `NDCubeBase` later whereas there is if we decide to go the other way.